### PR TITLE
Connecting toggleCollapsedSignal to handler right at creation of Mark…

### PR DESCRIPTION
**Pull request by schmidi314**
…downCell.

## References

#11442 

## Code changes

Connected the toggleCollapsedSignal of MarkdownCell to the handler in the _createMarkdownCell method.

### CODE REVIEW

File: packages/notebook/src/widget.ts [hunk 1]
```diff
@@ -647,6 +647,14 @@ export class StaticNotebook extends Widget {
     const cell = this.contentFactory.createMarkdownCell(options, this);
     cell.syncCollapse = true;
     cell.syncEditable = true;
+    // Connect collapsed signal for each markdown cell widget
+    if(this instanceof Notebook) {
```
Comment (fcollonval) [inline packages/notebook/src/widget.ts:651]: Could you elaborate on the need for that test? As the method is `private` it should not be required.

File: packages/notebook-extension/src/index.ts [hunk 2]
```diff
@@ -1518,6 +1518,7 @@ function addCommands(
           list: IObservableUndoableList<ICellModel>,
           args: IObservableList.IChangedArgs<ICellModel>
         ) => {
+          /* probably this can go away
```
Comment (fcollonval) [inline packages/notebook-extension/src/index.ts:1521]: Yes please delete it.

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @schmidi314 I get only two small comments on code style.

File: packages/notebook/src/widget.ts (hunk 1 above)
Comment (schmidi314) [reply packages/notebook/src/widget.ts:651]: Hm, seems this is a remnant of me trying to cast in type script. Sorry, new to this language. I'll fix that.

File: packages/notebook-extension/src/index.ts (hunk 2 above)
Comment (schmidi314) [reply packages/notebook-extension/src/index.ts]: Done.

**Comment by schmidi314** [comment]
@meeseeksdev tag bug

**Comment by krassowski** [comment]
@schmidi314 I think you accidentally uploaded `highlight-latex-jupyterlab-linux.png` instead of `do-not-highlight-not-latex-jupyterlab-linux.png`.

**Comment by krassowski** [comment]
Also, there are some pixel changes in the editor positioning in these tests - https://github.com/jupyterlab/jupyterlab/pull/11507 will fix it once ready, so no need to worry about it too much.

**Comment by schmidi314** [comment]
Ah, damn, sorry. Thanks for the hint.

OK, good to know about #11507. But let me fix it anyway, for practicing reasons :)

**Review by fcollonval — APPROVED** [review]
Thanks @schmidi314

**Comment by fcollonval** [comment]
@meeseeksdev please backport to 3.2.x

**Pull request merged**